lib/ace/mode/latex_parser.js

```diff
@@ -1,23 +1,42 @@
 define(function(require, exports, module) {
 "use strict";
 
-var WORD_PATTERN = "[a-zA-Zа-яА-ЯёЁ]+";
-var WORD_REGEX = new RegExp("^" + WORD_PATTERN);
+var WORD_REGEX = /^[a-zа-яё]+/i,
+    // % * <foo@bar.com> 2015-01-12T00:17:27.829Z:
+    COMMENT_REGEXP = /^\s*%\s([\*\^])\s\<(.+)\>\s(.+):$/;
```

Comment:
maybe rename it do DISCUSSION_COMMENT_REGEXP ? Because it is not just a generic comment.